src/main/java/com/mastercard/developer/utils/EncryptionUtils.java

```diff
@@ -32,16 +31,29 @@ private EncryptionUtils() {
     /**
      * Populate a X509 encryption certificate object with the certificate data at the given file path.
      */
-    public static Certificate loadEncryptionCertificate(String certificatePath) throws CertificateException, FileNotFoundException {
+    public static Certificate loadEncryptionCertificate(String certificatePath) throws CertificateException, IOException {
+        return loadEncryptionCertificate(Files.readAllBytes(Paths.get(certificatePath)));
+    }
+
+    /**
+     * Populate a X509 encryption certificate object with the certificate data at the given certificate data in bytes.
+     */
+    public static Certificate loadEncryptionCertificate(byte[] certificateBytes) throws CertificateException {
```

Comment:
For more flexibility we could use a `InputStream` instead of a `byte[]`. This way we can handle data from various sources, files, network, etc.

This is the choice that was made here: https://github.com/Mastercard/oauth1-signer-java/blob/main/src/main/java/com/mastercard/developer/utils/AuthenticationUtils.java#L28

Reply:
I agree, with that change, we still have a single interface and better flexibility